src/regtech_data_validator/validation_results.py

```diff
@@ -0,0 +1,19 @@
+import pandas as pd
+
+from dataclasses import dataclass
+from enum import StrEnum
+
+
+class ValidationPhase(StrEnum):
+    SYNTACTICAL = "Syntactical"
+    LOGICAL = "Logical"
+
+
+@dataclass
+class ValidationResults(object):
+    single_field_count: int
+    multi_field_count: int
+    register_count: int
+    is_valid: bool
+    findings: pd.DataFrame
+    phase: ValidationPhase = None
```

Comment:
let's get rid of the default placeholder now, also let's slap a `frozen=True` to the decorator since we aren't modifying it anymore.

Reply:
Very cool, I learned a thing.